pkg/kubectl/cmd/convert/import_known_versions.go

```diff
@@ -19,6 +19,9 @@ package convert
 // These imports are the API groups the client will support.
 // TODO: Remove these manual install once we don't need legacy scheme in convert
 import (
+	_ "k8s.io/kubernetes/pkg/apis/admission/install"
+	_ "k8s.io/kubernetes/pkg/apis/admissionregistration/install"
+	_ "k8s.io/kubernetes/pkg/apis/apiserverinternal/install"
```

Comment:
Why do we need to import the internal APIs?

Reply:
You are right, there is no need to import the internal.